### PR TITLE
[Images] Added API tokens page

### PR DESCRIPTION
**Pull request by marciocloudflare**
- Reworded Upload images section
- Added API token page
- Re-organized pages 
- Moved "Making your first API request" to its own page and explained where to find the info needed to start using the API

### CODE REVIEW

File: products/images/src/content/cloudflare-images/api-request.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+1. In the **Overview** app, scroll down to find your Account ID.
+
+</div>
+</details>
+
+<details>
+<summary>Your Global API Key or API Token</summary>
+<div>
+
+Refer to [API tokens](/cloudflare-images/upload-images/api-token).
```
Comment (pedrosousa) [inline products/images/src/content/cloudflare-images/api-request.md:24]: We already have very similar content... that we may need to update:
https://developers.cloudflare.com/api/tokens

File: products/images/src/content/cloudflare-images/api-request.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+</div>
+</details>
+
+Once you have this information, you are ready to upload your first image to Cloudflare Images using the API:
+
+```bash
+curl --request POST \
+  --url https://api.cloudflare.com/client/v4/accounts/:account_tag/images/v1 \
+  --header 'Authorization: Bearer :token' \
+  --form file=@./YOUR_IMAGE.jpeg
```
Comment (pedrosousa) [inline products/images/src/content/cloudflare-images/api-request.md]:
```suggestion
curl -X POST \
"https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/images/v1" \
-H 'Authorization: Bearer <API_TOKEN>' \
-F file=@./<YOUR_IMAGE>.jpg
```
Updating the `curl` command to follow our usual guidelines

File: products/images/src/content/cloudflare-images/api-request.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```
+
+A successful response will look similar to this:
+
+```json
+{
+  "result": {
+    "id": "083eb7b2-5392-4565-b69e-aff66acddd00",
+    "filename": "triceratops.jpeg",
```
Comment (pedrosousa) [inline products/images/src/content/cloudflare-images/api-request.md]:
```suggestion
    "filename": "triceratops.jpg",
```
I think we should use the usual file extension (`.jpg`).

File: products/images/src/content/cloudflare-images/upload-images/api-token.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account.
+1. Click **Images**.
+1. In the Images tab, click **Use API**.
+1. In **API token**, paste your API token.
+1. Input the name of the image you want to use in **Image File Name**.
+
+The **cURL command** message box will show you the complete command you can use. Below, is an example for a PNG image: 
+
+```bash
+curl -X POST -F file=@./YOUR_IMAGE.png -H "Authorization: Bearer <Your_API_Token" https://api.cloudflare.com/client/v4/accounts/2f3dc21d188ba781322b520bbde5fb5a/images/v1
```
Comment (pedrosousa) [inline products/images/src/content/cloudflare-images/upload-images/api-token.md]:
```suggestion
curl -X POST -F file=@./<YOUR_IMAGE>.png -H "Authorization: Bearer <API_TOKEN>" https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/images/v1
```

File: products/images/src/content/cloudflare-images/upload-images/api-token.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+order: 2
+pcx-content-type: how-to
+---
+
+# API token
```
Comment (pedrosousa) [inline products/images/src/content/cloudflare-images/upload-images/api-token.md:6]: Consider leaving out this page (or at least this first section about token creation) and point to the existing one in the `/api` docs, stating that the token must have at least the _Cloudflare Images_ permission at the account level.

File: products/images/src/content/cloudflare-images/upload-images/dashboard-upload.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+order: 1
+pcx-content-type: how-to
+title: Images dashboard
+---
+
+# Images dashboard upload
+
+To quickly upload a single image to your Cloudflare Images account, you can use the dashboard.
```
Comment (pedrosousa) [inline products/images/src/content/cloudflare-images/upload-images/dashboard-upload.md]:
```suggestion
Use the dashboard to quickly upload a single image to your Cloudflare Images account.
```

File: products/images/src/content/cloudflare-images/upload-images/index.md [hunk 7]
```diff
@@ -10,9 +10,13 @@ Cloudflare Images allows developers to upload images that cover a wide range of
 <TableWrap>
 
 | Upload Method          | When to use       |
-| ---------------------- | ----------------- |
-| Images Dashboard       | Quick, one time uploads where automation is not required. |
-| API Token              | The image file is stored on your computer with access to your API Tokens. |
-| Direct Creator Uploads | The end user uploads the images but does not have access to your API tokens. For example, when users upload content directly to your website or mobile app. |
+| -----------------------------| ----------------- |
+| Images dashboard	| Quick, one time uploads where automation is not required. |
+| API Token		| A preferred way to upload images from your back end services. |
```
Comment (pedrosousa) [inline products/images/src/content/cloudflare-images/upload-images/index.md]:
```suggestion
| API Token		| A preferred way to upload images from your back-end services. |
```

File: products/images/src/content/cloudflare-images/upload-images/index.md [hunk 8]
```diff
@@ -10,9 +10,13 @@ Cloudflare Images allows developers to upload images that cover a wide range of
...
+| Direct Creator Uploads | Let your users upload images to Cloudflare Images without exposing your API key or token. |
 
-</TableWrap>
\ No newline at end of file
+</TableWrap>
+
+Refer to the links below for more information on each topic.
```
Comment (pedrosousa) [inline products/images/src/content/cloudflare-images/upload-images/index.md]:
```suggestion
Refer to the links below for more information on each topic:
```

**Review by pedrosousa — APPROVED** [review]

**Review by sejoker — APPROVED** [review]

**Pull request merged**